src/jit/emit.cpp

```diff
@@ -2969,9 +2999,10 @@ emitter::instrDesc  * emitter::emitNewInstrCallInd(int        argCnt,
                                                    VARSET_VALARG_TP GCvars,
                                                    regMaskTP  gcrefRegs,
                                                    regMaskTP  byrefRegs,
-                                                   emitAttr   retSizeIn)
+                                                   emitAttr   retSizeIn
+                                                   FEATURE_MULTIREG_RET_ONLY_ARG(emitAttr secondRetSizeIn))
 {
-    emitAttr  retSize = retSizeIn ? EA_ATTR(retSizeIn) : EA_PTRSIZE;
+    emitAttr       retSize = (retSizeIn != EA_UNKNOWN) ? retSizeIn : EA_PTRSIZE;
 
     bool gcRefRegsInScratch = ((gcrefRegs & RBM_CALLEE_TRASH) != 0);
```

Comment:
Note that if RAX is a gcref/byref then InstrDesc will be marked as idRef because retSizeIn will be EA_GCREF or EA_BYREF.  If RDX is a gcref/byref then we construct a LargeCall and set second RetReg is a gc-type.